spec/system/candidate_interface/entering_details/degrees/candidate_entering_degrees_spec.rb

```diff
@@ -0,0 +1,203 @@
+require 'rails_helper'
+
+RSpec.feature 'Entering a degree' do
+  include CandidateHelper
+
+  scenario 'Candidate enters their degree' do
+    given_i_am_signed_in
+    and_i_visit_the_site
+    when_i_click_on_degree
+    then_i_see_the_undergraduate_degree_form
+
+    # Add degree type
+    when_i_choose_uk_degree
+    and_i_click_on_save_and_continue
+    then_i_see_validation_errors_for_degree_type
+    when_i_fill_in_the_degree_type
+    and_i_click_on_save_and_continue
+
+    # Add subject
+    then_i_can_see_the_degree_subject_page
+    when_i_click_on_save_and_continue
+    then_i_see_validation_errors_for_degree_subject
+    when_i_fill_in_the_degree_subject
+    and_i_click_on_save_and_continue
+
+    # Add institution
+    then_i_can_see_the_degree_institution_page
+    when_i_click_on_save_and_continue
+    then_i_see_validation_errors_for_degree_institution
+    when_i_fill_in_the_degree_institution
+    and_i_click_on_save_and_continue
+
+    # Add completion status
+    and_i_confirm_i_have_completed_my_degree
+
+    # Add grade
+    then_i_can_see_the_degree_grade_page
+    when_i_click_on_save_and_continue
+    then_i_see_validation_errors_for_degree_grade
+    when_i_select_the_degree_grade
+    and_i_click_on_save_and_continue
+
+    # Add years
+    then_i_can_see_the_start_and_graduation_year_page
+    when_i_click_on_save_and_continue
+    then_i_see_validation_errors_for_graduation_year
+    when_i_fill_in_the_start_and_graduation_year
+    and_i_click_on_save_and_continue
```

Comment:
I'd be tempted to collapse these 'Add steps' into single methods, removing the need for the comments, eg -

`when_i_enter_the_degree_type`
`and_i_enter_the_subject`
`and_i_enter_the_institution`

And so on. There's an aversion in some projects to adding assertions to "when" steps, but I think it's fine if their purpose is to check you're on the right page or you're seeing an appropriate error message while carrying out some kind of action. Often leads to more helpful error messages when specs break, too.